ap_src/ap_app/utils/user_profile.py

```diff
@@ -0,0 +1,15 @@
+# Models
```

Comment:
Remove this comment

Reply:
The comments were simply for my benefit during refactoring but I suppose they are no longer needed.

Reply:
Fixed with 717b45d